_layouts/topic.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+---
+{% capture content %}
+{% include references.md %}
+{{page.excerpt}}
+
+{{page.extended_summary}}
+
+{% if page.primary_sources and page.primary_sources != '' %}
+## Primary code and documentation
+
+{% for source in page.primary_sources %}
+  {%- if source.link contains 'http://' or source.link contains 'https://' -%}
+    {% capture reference %}[{{source.title}}]({{source.link}}){% endcapture %}
+  {%- else -%}
+    {% capture reference %}[{{source.title}}][{{source.link}}]{% endcapture %}
+  {%- endif -%}
+  - {{reference}}
+{% endfor %}
+
+{% endif %}
+
+{% if page.optech_mentions and page.optech_mentions != '' %}
+## Optech newsletter and website mentions
+
+{% capture /dev/null %}
+  {% assign references = '' %}
+
+  {% assign mentions = page.optech_mentions | sort: "date" | reverse %}
+  {% for mention in mentions %}
+    {% if mention.feature == true %}
+      {% assign bold='*' | append: '*' %}
+    {% else %}
+      {% assign bold='' %}
+    {% endif %}
+
+    {% capture reference %}- {{bold}}{{mention.date}} [{{mention.title}}]({{mention.url}}){{bold}}
+{% endcapture %}
+
+    {% assign references = references | append: reference %}
+  {% endfor %}
+{% endcapture %}
+{{references}}
+
+{% endif %}
+
+{% if page.see_also and page.see_also != '' %}
```

Comment:
This section seems incomplete. I think it should include a for loop like the *Primary code and documentation* and *Optech newsletter and website mentions* sections.